src/sagemaker_pytorch_container/deep_learning_container.py

```diff
@@ -0,0 +1,65 @@
+import json
+import logging
+import requests
+
+
+def _retrieve_instance_id():
+    """
+    Retrieve instance ID from instance metadata service
+    """
+    url = "https://169.254.169.254/latest/meta-data/instance-id"
+    response = requests_helper(url, timeout=0.01)
+    print(response.text)
```

Comment:
Not using `logging`?

Reply:
removing this print statement had added it to test